src/components/pages/Offers/Offer/OfferDetails/__specs__/OfferEdition.spec.jsx

```diff
@@ -462,23 +462,27 @@ describe('offerDetails - Edition', () => {
         await renderOffers({}, store)
 
         // then
-        expect(await screen.getAllByText('My edited offer')).toHaveLength(2)
+        const titleInput = document.getElementById('name')
+        expect(titleInput.value).toBe(editedOffer.name)
+        const titlePreviews = document.getElementsByClassName('title-preview')
+        expect(titlePreviews).toHaveLength(1)
+        expect(titlePreviews[0].firstChild.wholeText).toBe(editedOffer.name)
```

Comment:
Pourquoi avoir rajouté un test sur le fait que l'input était bien rempli du nom de l'offre ? 
Puisqu'on est dans un describe qui concerne l'aperçu de l'offre je pense qu'on peut se contenter de vérifier sa présence dans la section d'aperçu :
```suggestion
        expect(within(screen.getByTestId('offer-preview-section')).getByText(editedOffer.name)).toBeInTheDocument()
```

Reply:
Comme le test original vérifiait que le titre était présent 2 fois sur la page (dans le formulaire et la prévisualisation), je me suis contenté de vérifier la même chose avec l'input à la place du textarea. Mais tu as raison, vu qu'on est dans un test de la prévisualisation, on n'a pas à s'occuper de ce qu'il y a dans le formulaire.